### PR TITLE
Add testing/variables articles

### LINKED ISSUE #52: articles/testing/using-variables.md
* [x] overview
* [x] environment
* [x] context

### PR DESCRIPTION
**Pull request by rossmcdonald**
Fixes #52

### CODE REVIEW

File: articles/testing/variables-context.md [hunk 1]
```diff
@@ -1 +1,100 @@
...
+### In Scripts
+
+Similar to the example above, when referencing a context variable in a step
+script, use the following syntax:
+
+```javascript
+$.ctx.get('myVariable');
+```
+
+Where the curly brackets are absent, and we are using the `get()` method for
```
Comment (rowa97) [inline articles/testing/variables-context.md]: not sure what curly brackets you are referencing

File: articles/testing/variables-context.md [hunk 2]
```diff
@@ -1 +1,100 @@
...
+```
+
+Where:
+
+* `{...}` - Curly brackets signify that this is a variable reference. See the
+  [variables overview](./variables-overview.md) section for more information on
+  how variables are used.
+* `$` - The "single dollar sign" syntax is a reference to the current scenario's
+  "context".
+* `ctx` - Every scenario context has an `ctx` property that signifies this being
```
Comment (rowa97) [inline articles/testing/variables-context.md]: I'm not quite sure what you are trying to say here. Rework "this being" but also potentially simplify the whole statement. Also, it is indirect compared with the other definitions, what is 'ctx' actually doing? does it do anything other than signify it is a reference to the editor's environment?

File: articles/testing/variables-context.md (hunk 1 above)
Comment (rossmcdonald) [reply articles/testing/variables-context.md]: My bad, this was referring to the braces (`{}`) used above. I've updated this for clarity.

File: articles/testing/variables-context.md (hunk 2 above)
Comment (rossmcdonald) [reply articles/testing/variables-context.md]: Updated. I accidentally left some content from the environment variables article, which is (partly) why it was confusing.

**Pull request merged**